internal/osbuildexecutor/runner-common.go

```diff
@@ -36,30 +36,31 @@ func handleProgress(osbuildStatus *osbuild.StatusScanner, logger logrus.FieldLog
 		}
 		if st.Message != "" {
 			logger.WithFields(progress).Infof("OSBuild status: %s", st.Message)
-			if job != nil && time.Since(lastUpdated) > MinTimeBetweenUpdates {
-				lastUpdated = time.Now()
-				partial := worker.JobResult{
-					Progress: &worker.JobProgress{
-						Message: st.Message,
-					},
-				}
-				if st.Progress != nil {
-					partial.Progress.Done = st.Progress.Done
-					partial.Progress.Total = st.Progress.Total
-					// more than 1 level of subprogress is not expected, just
-					// pipelines and stages.
-					if st.Progress.SubProgress != nil {
-						partial.Progress.SubProgress = &worker.JobProgress{
-							Message: st.Progress.SubProgress.Message,
-							Done:    st.Progress.SubProgress.Done,
-							Total:   st.Progress.SubProgress.Total,
-						}
+			if job == nil || time.Since(lastUpdated) < MinTimeBetweenUpdates {
```

Comment:
(nitpick) might be nicer to move this change directly two commits earlier when this is indroduced but I have not looked deep, if there is a different reason to do it this way thats fine!